cargo-miri/bin.rs

```diff
@@ -739,16 +746,7 @@ fn phase_cargo_runner(binary: &Path, binary_args: env::Args) {
     let json_flag = "--json";
     while let Some(arg) = args.next() {
         if arg == extern_flag {
```

Comment:
This is now the only use of `extern_flag` so I guess it'd make sense to inline this.